src/plugins/intel_cpu/src/emitters/jit_eltwise_emitters.cpp

```diff
@@ -1958,8 +1955,214 @@ void jit_soft_sign_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, con
 }
 
 void jit_soft_sign_emitter::register_table_entries() {
-    push_arg_entry_of("one", 0x3f800000, true);
+    push_arg_entry_of("one", const_1F, true);
     push_arg_entry_of("positive_mask", 0x7fffffff, true);
 }
+
+/// IS_FINITE ///
+template <>
+void jit_is_finite_emitter::emit_isa<x64::avx512_core>(const std::vector<size_t> &inVecIdxs, const std::vector<size_t> &outVecIdxs) const {
+    auto vSrc = Zmm(inVecIdxs[0]);
```

Comment:
Minor:
I understand that we don't have exact style code at the moment. But all emitters have the same style: snake code style for classes, parameters and variables. I'd suggest to align code with this style.

Reply:
Modified